plugins/DigitalLibrary/src/DigitalFactoryApiClient.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, application: CuraApplication, on_error: Callable[[List[CloudE
         self._http = HttpRequestManager.getInstance()
         self._on_error = on_error
         self._file_uploader = None  # type: Optional[DFFileUploader]
+        self._library_max_private_projects: Optional[int] = None
```

Comment:
Aren't all of these projects private?
```suggestion
        self._library_max_projects: Optional[int] = None
```

Reply:
Nope. The ones retrieved by calling the API without filters contains both private and shared projects. We only check against the private (with query parameter `shared=false`).